src/components/login/MagicLinkInputs.tsx

```diff
@@ -88,6 +91,8 @@ const MagicLinkInputs = ({ onSubmit, schema, uiSchema }: Props) => {
 
             if (!hasToken) {
                 displayNotification('login.magicLink', 'success');
+            } else {
+                navigate(authenticatedRoutes.home.path, { replace: true });
```

Comment:
With the new routers we no longer get auto re-renders that would lead to auto navigating to this page.